### DISCUSSION TITLE
feature request: multiple rtmp endpoints with individualized audio and video controls

### DISCUSSION THREAD

**Discussion by endingwithali** [obsproject/obs-studio#10038] [Development Questions]
as title says.... hope to see this on the roadmap in the future <3

**Comment by Fenrirthviti** [comment]
This sounds more likely to be a support request, and we are not currently accepting support requests on GitHub Discussions. Make sure you read any [pinned posts](https://github.com/obsproject/obs-studio/discussions/5206) on discussion boards before opening a new discussion. Please use our [forums](https://obsproject.com/forum/) or [Discord](https://obsproject.com/discord/) for further assistance.

Thank you!
